src/main/java/de/xm/jdbcexcel/cellwriters/BooleanCellWriter.java

```diff
@@ -0,0 +1,13 @@
+package de.xm.jdbcexcel.cellwriters;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Workbook;
+
+public class BooleanCellWriter extends AbstractCellWriter<Boolean> {
+
+    @Override
+    protected int doWriteCell(Workbook workbook, Cell cell, Boolean cellValue) {
+        cell.setCellValue(cellValue);
+        return cellValue ? 4 : 5;
```

Comment:
this took me a minute, maybe some final constant or something like "TRUE".length() might be more readable. A single linke comment might also be ok.